dev/source/docs/ros2-gazebo.rst

```diff
@@ -40,7 +52,7 @@ Update ROS dependencies:
 .. code-block:: bash
 
     cd ~/ardu_ws
-    source /opt/ros/humble/setup.bash
+    source /opt/ros/$ROS_DISTRO/setup.bash
```

Comment:
Please revert. ROS_DISTRO is not set until AFTER you run this.
Consider doing it like `source /opt/ros/<your_ros_distro>/setup.bash`